dao/src/main/java/com/epam/esm/config/DaoConfig.java

```diff
@@ -1,44 +1,29 @@
 package com.epam.esm.config;
 
-import com.epam.esm.constants.ApplicationConstants;
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
-
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Profile;
-import org.springframework.context.annotation.PropertySource;
-import org.springframework.context.annotation.Bean;
-import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.jdbc.datasource.DataSourceTransactionManager;
-import org.springframework.transaction.PlatformTransactionManager;
-import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.context.annotation.Primary;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
 import javax.sql.DataSource;
 
-@Profile("dev")
 @Configuration
-@EnableTransactionManagement
-@PropertySource("classpath:hikari.properties")
 @ComponentScan("com.epam.esm")
+@EnableJpaAuditing
```

Comment:
Doesn't this annotation belong to the spring-data?

Reply:
You are right, i got rid of this annotation in config class. Commit caae742.